lib/src/layouts/yaru_compact_layout.dart

```diff
@@ -18,8 +21,8 @@ class YaruCompactLayout extends StatefulWidget {
     required this.length,
```

Comment:
both `YaruCompactLayout` _and_ `YaruPageController` have required `length`

Reply:
Is that a problem? What would be a better solution?
Flutter does this in a similar way with `TabController` and the `Tab*` widgets (except that the widgets get a list of children with a fixed length - here we use builder functions that take an index instead of lists, so we need the length)

Reply:
That's probably the best option - thanks